CHANGELOG.md

```diff
@@ -17,6 +17,16 @@
     cart. You can set it through the new `Spree::Config.variant_pricer_class` setting. This
     class also knows which `PricingOptions` class it cooperates with.
 
+    #### Order Merging on Sign In:
+
+    Previously a before filter added in `core/lib/spree/core/controller_helpers/order.rb`
+    would cause sql queries to be used on almost every request in the frontend.
+    If you do not use Solidus Auth you will need to hook into this helper and
+    call `set_current_order` where your user signs in. This merges incomplete
+    orders a user has going with their current cart.  If you do use
+    Solidus Auth you will need to make sure you use a current enough
+    version that includes this explicit call.
```

Comment:
Could someone correct me if I'm wrong but I was fairly certain that the order merging logic is all contained in Core and was just unnecessarily being called in the front end.  After this change the order merging should just happen on the sign in which should be handled by the authentication system a user chooses.  I will change it to clarify that calling `set_current_order` will cause order merging